Sources/Authentication/LocalAuthentication/LocalAuthService.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+import UIKit
+
+public enum AuthType {
+    case none
+    case face
+    case finger
```

Comment:
Me and naming again
can we change these to "touch" and "passcode"?

Reply:
sure :)

Reply:
Updated now if you could re-review, thanks!